hopsworks-common/pom.xml

```diff
@@ -355,6 +341,11 @@
       <artifactId>bijection-avro_2.12</artifactId>
       <version>0.9.6</version>
     </dependency>
+    <dependency>
+      <groupId>org.yaml</groupId>
```

Comment:
Why is this needed?

Reply:
https://github.com/logicalclocks/hopsworks/pull/299/files#diff-bb2605f3192c961ef2b7393814fcf244R225